src/build-info/usage/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+orbs:
+  build-info: circleci/build-info@volatile
+
+workflows:
+  info:
+    jobs:
+      - build-info/dump
```

Comment:
Is this config expected to work by itself?

Or are you saying to use it with `workflows` OR local `jobs`? I would make separate files in that case. Otherwise, I might try to copy/paste this entire config to test it.